cmds/bmp/update_state_command.go

```diff
@@ -101,20 +101,25 @@ func (cmd updateStateCommand) Execute(args []string) (int, error) {
 // Private Methods
 
 func (cmd updateStateCommand) isValidState(state string) bool {
-	validState := map[string]bool{
-		"bm.state.new":     true,
-		"bm.state.using":   true,
-		"bm.state.loading": true,
-		"bm.state.failed":  true,
-		"bm.state.deleted": true,
+	switch state {
```

Comment:
I think you can simplify this code by having many `case` statements and one `return true` statement?